src/frontend/src/App.css

```diff
@@ -43,5 +43,5 @@
 
 @font-face{
   font-family: text-security-disc;
-  src: url("https://raw.githubusercontent.com/noppa/text-security/master/dist/text-security-disc.woff");
+  src: url("assets/text-security-disc.woff") format("woff");
```

Comment:
Note that if you just grabbed the woff file from dist folder, it actually contains [an old version of the font](https://github.com/noppa/text-security/blob/master/dist/README.md) and is not updated anymore. Should still be fine I think, but the better way would be installing it from npm ([`text-security`](https://npm.im/text-security))